.github/workflows/main.yml

```diff
@@ -108,3 +107,90 @@ jobs:
         run: ./scripts/release_canary.sh
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Preparing tarballs of packages for testing templates installation
+        run: node ./scripts/build-tarballs.js
+
+      # TODO: in the future we want to extract this step into a separate job to be able
+      # to test this in multiple environments (macOS, Window) and with different node
+      # versions (10, 12). This can be done using the action strategy matrix.
+      # However, in order to do that, we need to be able to persist the tarballs between
+      # jobs. This can be done by using the upload/download artifacts actions.
+      # However, at this time there is no option to clean up the artifacts automatically
+      # after the workflow ends, and the artifacts will stay for 90 days.
+      # Some related issues:
+      # - https://github.com/actions/upload-artifact/issues/5
+      # - https://github.com/actions/upload-artifact/issues/45
+      - name: Installing and building template starter
+        run: node ./scripts/install-template.js
+        env:
+          TEMPLATE_NAME: starter
+
+  #     - name: Uploading tarball artifacts
+  #       uses: actions/upload-artifact@v1
+  #       with:
+  #         name: dist-tarballs
+  #         path: dist-tarballs
+
+  # install_template_starter:
+  #   needs: build_lint_and_test
+
+  #   runs-on: ${{ matrix.os }}
+
+  #   strategy:
+  #     matrix:
+  #       os: [macos-latest, windows-latest]
+  #       node: [10, 12]
+
+  #   steps:
+  #     - name: Checkout
+  #       uses: actions/checkout@v2
+
+  #     - uses: actions/download-artifact@v1
+  #       with:
+  #         name: dist-tarballs
+  #         path: dist-tarballs
+
+  #     - name: Setup node
+  #       uses: actions/setup-node@v1
+  #       with:
+  #         node-version: ${{ matrix.node }}
+
+  #     - name: Installing shelljs globally
+  #       run: yarn add global shelljs
+
+  #     - name: Installing and building template starter
+  #       run: node ./scripts/install-template.js
+  #       env:
+  #         TEMPLATE_NAME: starter
+
+  #     - name: Fetching workflow artifacts
```

Comment:
This was just an attempt to "clean up" the artifacts, but it's a bit more complicated that this.
It's cool though to be able to make API requests using actions and to parse the results.

https://github.com/octokit/request-action
https://github.com/gr2m/get-json-paths-action

Reply:
Wow. This is really dope!